scripts/apps/authoring/authoring/directives/SendItem.js

```diff
@@ -26,6 +26,10 @@ export function SendItem($q, api, desks, notify, authoringWorkspace,
         controllerAs: 'vm',
         templateUrl: 'scripts/apps/authoring/views/send-item.html',
         link: function sendItemLink(scope, elem, attrs, ctrl) {
+            // use the editor service of editor3, if it's active globally, or on the content profile body
+            const isEditor3Body = authoring.editor && authoring.editor.body_html && authoring.editor.body_html.editor3;
+            const editor = config.features.onlyEditor3 || isEditor3Body ? editor3 : editor2;
```

Comment:
imo this should go into some editor factory not to duplicate this logic in different places